### DISCUSSION TITLE
Font sizes are not rendering correctly in Chrome

### DISCUSSION THREAD

**Discussion by MarcyLina** [tailwindlabs/tailwindcss#14104] [Help]
This very well may be a Chrome issue. I'm on version 127 and macOS Sonoma.

The problem I've run into is that nothing smaller than `text-base` will render. Anything I have with the classes `text-sm` and `text-xs` show up as 1rem. Oddly the line-height _does_ work.

On my main page, I added some `<p>` tags to test other browsers using `text-xs`, `text-sm`, `text-base,` and `text-lg`. The screenshots are the results I got:

**Chrome:**
<img width="436" alt="image" src="https://github.com/user-attachments/assets/6f4829ae-cce0-4ac1-b574-a906f2fe9667">

**Firefox and Safari:**
<img width="380" alt="image" src="https://github.com/user-attachments/assets/08c46cbc-8a4d-4164-8087-5bb4e04d404c">

I'm on a Laravel, Inertia, and Vue 3 build. All of those, and Tailwind are on the latest versions.

I did see this issue  [#1402](https://github.com/tailwindlabs/tailwindcss/issues/1402) while searching for answers, which is fairly similar regarding font-weight and it turned out to be a Chrome bug, so I figured I'd post anyway to at least get some attention on the issue and perhaps have it reported to the folks at Chrome.
</div>

**Comment by adamwathan** [comment]
Do you have a "Minimum font size" configured in Chrome?

<img width="707" alt="image" src="https://github.com/user-attachments/assets/dc6f2451-d6b4-46de-a911-875522a8c1d1">

In this reproduction I see the correct font sizes in Chrome:

https://play.tailwindcss.com/sCIggoj9Qf

**Reply by MarcyLina** [reply]
My chrome settings are the same as you pictured

**Reply by adamwathan** [reply]
Can you reproduce the issue in a [Tailwind Play](https://tailwind.new/)? Does it happen on every computer you've tried it on?

**Reply by MarcyLina** [reply]
Out of desperation, I completely uninstalled Chrome and reinstalled it. That seemed to do the trick 🧐. Thank you for looking!
